vue-components/tests/e2e/custom-commands/openComponentStory.js

```diff
@@ -3,13 +3,9 @@ module.exports.command = function openComponentStory( component ) {
 		.click( `a[href="#${component}"]` )
 		.element(
```

Comment:
Yay for deleting code!! This can become even smoler!
```js
	return this.waitForElementVisible( '#root' )
		.click( `a[href="#${component}"]` )
		.frame( 'storybook-preview-iframe' );
```

Reply:
I just remembered what the problem was with the short version. Apparently Geckodriver has a problem with iframes and the tests on firefox fail if we don't invoke the iframe in a callback. 
https://github.com/nightwatchjs/nightwatch/issues/2030

Reply:
> Geckodriver has a problem with iframes

Can't blame it, iframes are weird. Alrighty, then this is fine as it is.